Ports/lua/package.sh

```diff
@@ -3,5 +3,8 @@ port=lua
 version=5.3.6
 files="http://www.lua.org/ftp/lua-${version}.tar.gz lua-${version}.tar.gz fc5fd69bb8736323f026672b1b7235da613d7177e72558893a0bdcd320466d60"
 auth_type=sha256
-makeopts=("-j$(nproc)" "serenity" "CC=${CC}" "AR=${AR}" "RANLIB=${RANLIB}")
```

Comment:
I believe `make` has a `-C <director>` flag which first goes into the directory and then executes the commands. That means we don't have to add the build function here.

Reply:
I don't quite know why this is the case but seems that it doesn't like the space in there. Perhaps one of `-C${port}-${version}/src` or `-Csrc/` does work? 
And otherwise this is fine.

Reply:
Alright, I fixed up the commit, it works and builds now.

Reply:
Nice! (are you sure this is the correct commit? It looks the same on my end)

Reply:
Oops. I forgot to `git add`. Sorry about that.